packages/matchbox/src/components/Select/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as Select } from './Select';
+export * from './Select';
+export * from './Options';
+export * from './Option';
```

Comment:
Do we need to export the types? We don't expose the `Option` and `Options` components to end users